extensions/blocks/podcast-player/closest-shim.js

```diff
@@ -0,0 +1,21 @@
+// Shim for Element.prototype.closest
+// https://developer.mozilla.org/en-US/docs/Web/API/Element/closest
+
+if ( ! Element.prototype.matches ) {
+	Element.prototype.matches =
+		Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
+}
+
+if ( ! Element.prototype.closest ) {
+	Element.prototype.closest = function( s ) {
+		let el = this;
+
+		do {
+			if ( el.matches( s ) ) {
+				return el;
+			}
+			el = el.parentElement || el.parentNode;
+		} while ( el !== null && el.nodeType === 1 );
+		return null;
+	};
+}
```

Comment:
Should we put this file somewhere it can be shared with other blocks? E.g. https://github.com/Automattic/jetpack/tree/master/extensions/shared